app/src/main/assets/web_extensions/youtube_webcompat/main.css

```diff
@@ -1,3 +1,6 @@
-div.alert-with-button-renderer {
-	display:none;
+/* To hide harmless warnings. */
+.alert-with-button-renderer, .ytm-alert-with-button-renderer, ytm-alert-with-button-renderer,
+.ytp-ad-module, ytp-ad-module
```

Comment:
oops, missing comma (regressed issue #837 and #1056)

Reply:
fixed in PR #1217: https://github.com/MozillaReality/FirefoxReality/pull/1217/files#diff-ebdf9c97fadbb79c463a9b4302a68491R3